docs/specs/resolve.yml

```diff
@@ -168,17 +168,6 @@ outputs:
   docs/a.json: |
     {"content":"<p><a href=\"/absolute/path?a#b\">redirect</a></p>"}
 ---
-# Files redirected to a relative URL
```

Comment:
this case should be added by reason?

Reply:
impact test will test us it's used or not :)

Reply:
there does exist urls other than `docs.microsoft.com`.
```text
articles/machine-learning/service/aml-pricing.md: 'https://azure.microsoft.com/pricing/details/machine-learning-services'
articles/machine-learning/studio/live-chat.md: 'https://social.msdn.microsoft.com/Forums/en-US/home?forum=MachineLearning'
```
how should we handle this?

Reply:
these are simple redirections (without affecting document id)